### ISSUE TITLE
Invalid version constraint in module v0.3.3

### ISSUE THREAD

**Issue by oleksiypavlenko** [spotinst/terraform-spotinst-gcp-connect#14]
**What happened**:

```
Initializing the backend...
╷
│ Error: Invalid version constraint
│ 
│   on .terraform/modules/spotinst/versions.tf line 4, in terraform:
│    4:     google = {
│    5:       version = ">=v4.0.0"
│    6:     }
│ 
│ Incorrect version constraint syntax: invalid specification ">=": only one space is expected after the operator ">=".
```

**What you expected to happen**:

Successful initialization

**How to reproduce it (as minimally and precisely as possible):**

Use the version v0.3.3 module and perform `terraform init`

**Anything else we need to know:**

**Environment:**

- Terraform version: v1.2.5
- Module version: 0.3.3
- OS: N/A
- Kernel: N/A
- Others: N/A

**Comment by oleksiypavlenko** [comment]
Looks like this [may be already addressed](https://github.com/spotinst/terraform-spotinst-gcp-connect/pull/13/files) in v0.3.4 of module

**Comment by stevenfeltner** [comment]
Correct, there was a typo in that version and was corrected in version v0.3.4
